document/chapters/02-background.tex

```diff
@@ -696,7 +694,7 @@ \subsection{Octrees}%%Status:Done%%
 KD-trees. These traversals are conceptually similar to that of the KD-tree's but
 typically employ some form of Morton encoding to determine which node in the
 octree the ray should visit next \cite{Revelles_2000}. Other traversal
-techniques allow the octree to avoid creating and traversing nodes containing
+techniques allow the Octree to avoid creating and traversing nodes containing
```

Comment:
is it necessary/correct to capitalize Octree?

Reply:
Looked this up. It is not. I think in older papers it went back and forth, but the community has settled on lower case recently. Updating...